### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by PwrSrg** [sveltejs/template#287]
### Describe the bug

When testing Svelte for the very first time, it throws an error.

![image](https://user-images.githubusercontent.com/711603/159505391-9cbffb83-4737-486b-b401-add6d52d5f9d.png)

### Reproduction

npx degit sveltejs/template my-svelte-project
cd my-svelte-project
npm install
npm run dev

### System Info

```shell
System:
    OS: Windows 10 10.0.22000
    CPU: (12) x64 AMD Ryzen 5 3600X 6-Core Processor
    Memory: 17.89 GB / 31.93 GB
  Binaries:
    Node: 17.7.2 - C:\Program Files\nodejs\node.EXE
    npm: 8.5.2 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22000.120.0), Chromium (96.0.1054.57)
    Internet Explorer: 11.0.22000.120
```

### Severity

blocking all usage of svelte

**Comment by PwrSrg** [comment]
After further debugging, the issue is somehow related to running from a **mapped network drive**.

I will update this post as I discover more.

**Comment by jweathers** [comment]
I'm having this issue as well.  Running node 17 on windows 10.  No mapped drives are used in this project.  I do not have any mapped drives.

**Comment by Conduitry** [comment]
I've just tested this again on a fresh Windows 10 sandbox, Node 16.13.2, and cannot reproduce it.

**Comment by mscrocdile** [comment]
I can confirm it works in sandbox.
I can fix that problem if i start "npm run dev" with admin access rights. No idea why.

**Comment by jasonlyu123** [comment]
Maybe it's related to https://github.com/rollup/rollup/issues/4439. Can you check if your path has a different casing between the terminal and the file system?

**Comment by Chaciej** [comment]
fresh install, step by step from the site instructions, what's happening?
![svelte not working](https://user-images.githubusercontent.com/18661408/163064638-ffe91b3f-3e84-42b1-927c-ad7d7f84104b.png)

well... it did install on a second drive and it somehow works
so the issue is with C drive or something with permissions... dunno

**Comment by DonTomato** [comment]
> so the issue is with C drive or something with permissions... dunno

I tried, and it fails with this error on both C, D, etc. I couldn't find such a place where it works.

**Comment by DonTomato** [comment]
@jasonlyu123 Yes, that's it. As soon as I changed `c:\sources\...` for `C:\sources\...` in my terminal, it started to work. 
This is just an insanely stupid bug 🤡
